koku/api/settings/test/tag_mappings/test_view.py

```diff
@@ -0,0 +1,288 @@
+import json
+import random
+
+from django.urls import reverse
+from django_tenants.utils import tenant_context
+from faker import Faker
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.test import APIClient
+
+from api.iam.test.iam_test_case import IamTestCase
+from api.settings.tag_mappings.query_handler import format_tag_mapping_relationship
+from api.settings.tag_mappings.view import SettingsTagMappingFilter
+from reporting.provider.all.models import EnabledTagKeys
+from reporting.provider.all.models import TagMapping
+
+
+class SettingsTagMappingViewTestCase(IamTestCase):
```

Comment:
I think usually our test case names are `Test{Classname}`, so I think this test class should be:
```suggestion
class TestSettingsTagMappingView(IamTestCase):
```